app/Post.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Post extends Model
+{
+    private $title;
+    private $content;
+
+    public function user() 
+    {
+        return $this->hasOne('App\User', 'user_id');
+
+    }
+
+    public function getTitle()
+    {
+        return $this->title;
+    }
+
+    public function setTitle($value)
+    {
+        $this->title = $value;
+    }
```

Comment:
ovako neces sigurno setovati title jer treba da ti se perzistuje sa bazom, tako da ovako smjestanje ti nece imati neku ulogu, uglavnom ces odraditi nad Eloquent modelom `create` i on ce ti na kraju vratiti citav model

Reply:
Prilikom kreiranja inserta u bazi preko ovog Eloquent modela-a , tebi laravel nece dozvoliti snimanje vrijednosti svih atributa  odjednom, pa onda moras imati `fillable ` ILI `guarded ` . 

u `fillable` stavljas sve one kolone koje ces moci upisivati preko modela u bazu i to ti je jedan moguci nacin 
ili ces korsitis `guraded` i stavljati sve kolomne koje neces moci omoguciti da se snimaju u bazi.

Znaci koristi ili jedno ili drugo, NIKAD oba, a samim tim kad iskoristis jedno, za ostala polja koja ne navedes u tom jednom bice pretpostavljeno da mislis da ne treba da se upsiuju/neupisuju u bazu